### PR TITLE
Fixes #185028

### LINKED ISSUE #185028: add accessible help menu for diff editor
We can likely just tweak the current one for the editor

This would allow us to replace the hint about `shift+f7` with a help menu hint and we could teach users what to expect wrt audio cues etc.

Blocked on the diff editor refactor

### PR DESCRIPTION
**Pull request by hediet**
Fixes #185028

### CODE REVIEW

**Comment by meganrogge** [comment]
When I test, the `editor` is overriding this. 

Also, I forgot to mention, but you'll want to add a hint to the aria label so that this is discoverable. You'll also want to add an associated `accesssibility.verbosity` setting so the hint can be disabled.

**Comment by meganrogge** [comment]
You'll also want to add an `accessibility.verbosity` setting similar to what is done for the other features so that users discover this menu

File: src/vs/editor/browser/widget/diffEditorWidget2/diffEditorEditors.ts [hunk 1]
```diff
@@ -149,7 +151,10 @@ export class DiffEditorEditors extends Disposable {
 	}
 
 	private _updateAriaLabel(ariaLabel: string | undefined): string | undefined {
-		const ariaNavigationTip = localize('diff-aria-navigation-tip', ' use Shift + F7 to navigate changes');
+		if (!ariaLabel) {
+			ariaLabel = '';
+		}
+		const ariaNavigationTip = localize('diff-aria-navigation-tip', ' use {0} to open the accessibility help.', this._keybindingService.lookupKeybinding('editor.action.accessibilityHelp')?.getAriaLabel());
```
Comment (meganrogge) [inline src/vs/editor/browser/widget/diffEditorWidget2/diffEditorEditors.ts:157]: I think you want to check here if `accessibility.verbosity.diffEditor` is enabled?

Also, I'm not hearing it via the screen reader when I open a diff editor. are you?

**Comment by meganrogge** [comment]
using diff editor v2 and still not seeing the right help menu - are you?

Screenshot 2023-07-11 at 1 12 10 PM

**Comment by hediet** [comment]
> using diff editor v2 and still not seeing the right help menu - are you?

You have to reload the window after changing the setting unfortunately.

**Comment by meganrogge** [comment]
Perhaps we should also mention the audio cues that can be configured for this?

Screenshot 2023-07-11 at 1 46 58 PM

**Comment by meganrogge** [comment]
Yeah, but it's possible they will find some annoying, so we should let them know they can be disabled

File: src/vs/editor/browser/widget/diffEditorWidget2/diffEditorEditors.ts (hunk 1 above)
Comment (meganrogge) [reply src/vs/editor/browser/widget/diffEditorWidget2/diffEditorEditors.ts:157]: @hediet not sure if you missed these two remarks

Comment (hediet) [reply src/vs/editor/browser/widget/diffEditorWidget2/diffEditorEditors.ts:157]: It works when the diff editor is open and you change the setting.
This has been an issue in the old diff editor as well, I have to look into this seperately.

**Review by meganrogge — DISMISSED** [review]
thanks 👍🏼

**Review by joaomoreno — APPROVED** [review]

**Pull request merged**